### PR TITLE
Updating startup files for tests fixtures

### PR DESCRIPTION
**Pull request by imalberto**
Updating startup files for tests fixtures.

### CODE REVIEW

File: lib/middleware.js [hunk 1]
```diff
@@ -32,19 +37,8 @@ For example for using a custom error handler:
 
     app.use(mojito.middleware['mojito-handler-static']);
     app.use(mojito.middleware['mojito-contextualizer']);
-    app.use(require('./middleware/my-error-handler')());
-
-For application specific middleware, the `module.exports` object should 
-return a function that matches the `express` middleware signature.
+    app.use(require('./middleware/my-error-handler'));
```
Comment (caridy) [inline lib/middleware.js]: I think we should use an inline `function` instead of a require here.

File: tests/fixtures/gsg5-appConfig/app.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+app.get('/status', function (req, res) {
+    res.send('200 OK');
+});
+
+app.listen(app.get('port'), function () {
+    debug('Server listening on port ' + app.get('port') + ' ' +
+               'in ' + app.get('env') + ' mode');
+});
+
+module.exports = app;
```
Comment (caridy) [inline tests/fixtures/gsg5-appConfig/app.js]: breakline

File: tests/fixtures/store/app.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+app.get('/status', function (req, res) {
+    res.send('200 OK');
+});
+
+app.listen(app.get('port'), function () {
+    debug('Server listening on port ' + app.get('port') + ' ' +
+               'in ' + app.get('env') + ' mode');
+});
+
+module.exports = app;
```
Comment (caridy) [inline tests/fixtures/store/app.js]: :) break

File: tests/func/applications/frameworkapp/configapp/app.js [hunk 4]
```diff
@@ -38,3 +38,5 @@ app.listen(app.get('port'), function () {
     debug('Server listening on port ' + app.get('port') + ' ' +
                'in ' + app.get('env') + ' mode');
 });
+
+module.exports = app;
```
Comment (caridy) [inline tests/func/applications/frameworkapp/configapp/app.js]: one more

File: tests/func/applications/frameworkapp/routing/app.js [hunk 5]
```diff
@@ -49,3 +49,5 @@ app.listen(app.get('port'), function () {
     debug('Server listening on port ' + app.get('port') + ' ' +
                'in ' + app.get('env') + ' mode');
 });
+
+module.exports = app;
```
Comment (caridy) [inline tests/func/applications/frameworkapp/routing/app.js]: more

File: tests/func/applications/performanceapp/part1/app.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+app.get('/status', function (req, res) {
+    res.send('200 OK');
+});
+
+app.listen(app.get('port'), function () {
+    debug('Server listening on port ' + app.get('port') + ' ' +
+               'in ' + app.get('env') + ' mode');
+});
+
+module.exports = app;
```
Comment (caridy) [inline tests/func/applications/performanceapp/part1/app.js]: more

File: tests/func/applications/performanceapp/part2/app.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+app.get('/status', function (req, res) {
+    res.send('200 OK');
+});
+
+app.listen(app.get('port'), function () {
+    debug('Server listening on port ' + app.get('port') + ' ' +
+               'in ' + app.get('env') + ' mode');
+});
+
+module.exports = app;
```
Comment (caridy) [inline tests/func/applications/performanceapp/part2/app.js]: I know you like them!

File: tests/func/applications/performanceapp/part3/app.js [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+app.get('/status', function (req, res) {
+    res.send('200 OK');
+});
+
+app.listen(app.get('port'), function () {
+    debug('Server listening on port ' + app.get('port') + ' ' +
+               'in ' + app.get('env') + ' mode');
+});
+
+module.exports = app;
```
Comment (caridy) [inline tests/func/applications/performanceapp/part3/app.js]: is this the last one?

File: tests/func/applications/performanceapp/part4/app.js [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+app.get('/status', function (req, res) {
+    res.send('200 OK');
+});
+
+app.listen(app.get('port'), function () {
+    debug('Server listening on port ' + app.get('port') + ' ' +
+               'in ' + app.get('env') + ' mode');
+});
+
+module.exports = app;
```
Comment (caridy) [inline tests/func/applications/performanceapp/part4/app.js]: oh no!

File: tests/func/applications/performanceapp/part7/app.js [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+app.get('/status', function (req, res) {
+    res.send('200 OK');
+});
+
+app.listen(app.get('port'), function () {
+    debug('Server listening on port ' + app.get('port') + ' ' +
+               'in ' + app.get('env') + ' mode');
+});
+
+module.exports = app;
```
Comment (caridy) [inline tests/func/applications/performanceapp/part7/app.js]: lol

File: tests/func/applications/performanceapp/part6/app.js [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+app.get('/status', function (req, res) {
+    res.send('200 OK');
+});
+
+app.listen(app.get('port'), function () {
+    debug('Server listening on port ' + app.get('port') + ' ' +
+               'in ' + app.get('env') + ' mode');
+});
+
+module.exports = app;
```
Comment (caridy) [inline tests/func/applications/performanceapp/part6/app.js]: lol

File: tests/func/applications/performanceapp/part5/app.js [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+app.get('/status', function (req, res) {
+    res.send('200 OK');
+});
+
+app.listen(app.get('port'), function () {
+    debug('Server listening on port ' + app.get('port') + ' ' +
+               'in ' + app.get('env') + ' mode');
+});
+
+module.exports = app;
```
Comment (caridy) [inline tests/func/applications/performanceapp/part5/app.js]: ok ok, this is the last one! feel free to ignore my annoying comments.

**Comment by caridy** [comment]
+1 merge after updating the middleware docs and all the breaklines :)

**Comment by imalberto** [comment]
All builds passed.

**Pull request merged**